experimental/linting_tool/lib/app.dart

```diff
@@ -19,10 +19,10 @@ class LintingTool extends StatefulWidget {
   static const String homeRoute = routes.homeRoute;
 
   @override
-  _LintingToolState createState() => _LintingToolState();
+  LintingToolState createState() => LintingToolState();
```

Comment:
`State<LintingTool> createState() => _LintingToolState();`

Reply:
Updated all instances in this PR